_data/harnesses/change-topic-partitions-replicas/ksql.yml

```diff
@@ -0,0 +1,139 @@
+          stdout:
+            directory: tutorial-steps/dev/outputs
+
+    - title: Describe the new topic
+      content:
+        - action: execute
+          file: tutorial-steps/dev/describe-new-topic.sh
+          stdout: tutorial-steps/dev/outputs/actual-describe-from-topic2.txt
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/describe-new-topic.adoc
+
+    - title: Write your statements to a file
+      content:
+        - action: make_file
+          file: src/statements.sql
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/dev/make-src-file.adoc
+
+test:
+  steps:
+    - title: Create the test data
+      content:
+        - action: make_file
+          file: test/input.txt
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/test/make-test-input.adoc
+
+    - title: Produce the test data to the original topic
+      content:
+        - action: execute
+          file: tutorial-steps/test/harness-console-producer.sh
+          stdin: test/input.txt
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/test/produce-data-input.adoc
+
+        - name: wait for producer to write records
+          action: sleep
+          ms: 2000
+          render:
+            skip: true
+
+    - title: View the data in the original topic (partition 0)
+      content:
+        - action: execute
+          file: tutorial-steps/test/harness-console-consumer-topic1-partition-0.sh
+          stdout: tutorial-steps/test/outputs/actual-output-topic1-partition-0.txt
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/test/consume-data-topic1-partition-0.adoc
+
+    - title: View the data in the new topic (partition 0)
+      content:
+        - action: execute
+          file: tutorial-steps/test/harness-console-consumer-topic2-partition-0.sh
+          stdout: tutorial-steps/test/outputs/actual-output-topic2-partition-0.txt
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/test/consume-data-topic2-partition-0.adoc
+
+    - title: View the data in the new topic (partition 1)
+      content:
+        - action: execute
+          file: tutorial-steps/test/harness-console-consumer-topic2-partition-1.sh
+          stdout: tutorial-steps/test/outputs/actual-output-topic2-partition-1.txt
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/test/consume-data-topic2-partition-1.adoc
+
+prod:
+  steps:
+    - title: Send the statements to the REST API
+      content:
+        - action: skip
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/prod/submit-to-api.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/clean-up.sh
+          render:
+            skip: true
```

Comment:
`skip` needs to be replaced with `file:<path to adoc file>` to show the clean-up command to users.

Reply:
Ack, replacing now